packages/lesswrong/server/legacy-redirects/routes.ts

```diff
@@ -328,10 +328,6 @@ if (forumTypeSetting.get() === "AlignmentForum") {
     return makeRedirect(res, `/feed.xml`);
   })
   
-  addStaticRoute('/drafts', (params, req, res, next) => {
```

Comment:
Removes the static redirect from `/drafts` to `/account` - we should only delete this if we do actually add the new drafts page

Reply:
Has been promoted to a full-fledged route in `routes.ts`, so go ahead and delete this redirect